packages/generator/src/generator.js

```diff
@@ -205,7 +205,7 @@ export default class Generator {
         _generate: true,
         payload
       })
-      html = res.html
+      ;({ html } = res)
```

Comment:
Is outer semi required?

Reply:
It is required because of `})`. :relaxed: 

Otherwise eslint would complain too